docs/dev/data_validation.rst

```diff
@@ -0,0 +1,45 @@
+===============================================================================
+Data Validation
+===============================================================================
+The data validation tests are organized into datasource specific modules under
+``test/validate``. They test the quality and internal consistency of the data
+that is output by the PUDL ETL pipeline. Currently they only work on the full
+dataset, and do not have a ``--fast`` option. While it is possible to run the
```

Comment:
I think you removed the `--fast` flag from the other tests, so it doesn't really make sense here.